### ISSUE TITLE
auth0 with auth0-lock 10

### ISSUE THREAD

**Issue by saschanm** [SpoonX/aurelia-authentication#261]
Any plans to make auth0 provider compatible with version 10 of auth0-lock? There are breaking changes from version 9 to 10 and version 9 no longer being updated.

https://auth0.com/docs/libraries/lock/v10/migration-guide

**Comment by doktordirk** [comment]
@nexbit did that part. so, it's probably better if you auth0lock users make a PR to that end

**Comment by pfurini** [comment]
I already done the migration in my fork, but didn't have spare time to make a PR..

The main problem is that now is working great with v10, but breaks compatibility with v9.
So if anyone is using this and not willing to update, their app will break if they update the dependency.

**Comment by doktordirk** [comment]
can't be helped i'd say. i'd rather do that change now. we want to get of the rc train this week. a breaking change just after releasing would e more annoying

**Comment by pfurini** [comment]
@saschanm @doktordirk Done... CI build is in progress, then if someone can test this, it' be great! Doc update is pending, no spare time today

**Comment by pfurini** [comment]
Ok.. blocked by eslint, don't know why didn't failed on my side.. I'll fix that later

**Comment by doktordirk** [comment]
we've extended aurelia's linting rules last week slightly https://github.com/SpoonX/spoonx-tools/blob/master/.eslintrc.json
😄

**Comment by doktordirk** [comment]
@saschanm tx to @nexbit next update it'll be auth0-lock 10

**Comment by saschanm** [comment]
Thanks @nexbit and @doktordirk...love your work!
